modules/tinymce/src/themes/silver/main/ts/ui/general/OuterContainer.ts

```diff
@@ -75,6 +83,8 @@ interface ToolbarApis {
 }
 
 const factory: UiSketcher.CompositeSketchFactory<OuterContainerSketchDetail, OuterContainerSketchSpec> = (detail, components, _spec) => {
+  let state = false;
```

Comment:
This state seems specialised to toggling the toolbar drawer when switching views. Should we name it something more specific?

Reply:
Change this to `toolbarDrawerOpenState` couldn't figure out a better name.